src/main/java/com/google/graphgeckos/dashboard/fetchers/buildbot/BuildBotClientPopulation.java

```diff
@@ -0,0 +1,40 @@
+package com.google.graphgeckos.dashboard.fetchers.buildbot;
+
+import org.springframework.lang.NonNull;
+import java.util.Arrays;
+import java.util.Objects;
+
+/** BuildBot fetchers creator. Create a population of different fetchers with common baseUrl. */
```

Comment:
This responsibility would have been better delivered using the Factory pattern. Consider changing this.

Reply:
I don't think the factory pattern would be better than the existing one. I prefer leaving it as it is.